pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -107,8 +107,7 @@ var _ backupManifestFileIterator = &sstFileIterator{}
 
 // createIntroducedSpanFrontier creates a span frontier that tracks the end time
 // of the latest incremental backup of each introduced span in the backup chain.
-// See ReintroducedSpans( ) for more information. Note: this function assumes
-// that manifests are sorted in increasing EndTime.
+// Note: this function assumes that manifests are sorted in increasing EndTime.
```

Comment:
i believe we can remove this IntroducedSpanFrontier from restore as well, but perhaps we can deal with that in a follow up. I need to think a little more about the fact that 23.2 backups will still have reintroduced spans.